tests/integration/components/conversations/conversation-part-closed-test.js

```diff
@@ -34,16 +35,17 @@ test('if current user closes message, "You closed this" is rendered', function(a
     id: 1,
     username: 'testuser'
   };
-
-  let closedAt = 'MM/DD/YYYY hh:mm:ss';
-
   set(this, 'author', user);
-  set(this, 'closedAt', moment().fromNow());
-  stubService(this, 'current-user', { user: { id: 1 } });
+  stubService(this, 'current-user', { user });
+
+  let twoMinutesAgo = moment().subtract(2, 'minutes');
+  let twoMinutesAgoFriendly = twoMinutesAgo.from();
+  set(this, 'closedAt', twoMinutesAgo);
 
   renderPage();
-  assert.equal(page.closedAt.text, 'You closed this on', 'The closed at timestamp is rendered');
 
+  let text = `You closed this ${twoMinutesAgoFriendly}`;
+  assert.equal(page.closedAt.text, text, 'The closed at timestamp is rendered');
```

Comment:
Good thinking with this. Easier to read than `indexOf`.

A further suggestion would be to name it `expectedText`, so you'd have

```Javascript
let expectedText = //...
assert.equal(page.closedAt.text, expectedText, //...
```